[TASK] Always use PHPUnit 9

linawolf opened the pull request:
Remove deprecations in preparations for PHPUnit 10

linawolf committed:
[TASK] Always use PHPUnit 9

Remove deprecations in preparations for PHPUnit 10

linawolf committed:
[TASK] Drop Support for PHP <= 7.3

They are not compatible with PHPUnit 9 and outdated

brotkrueml reviewed (changes requested):
Please also migrate the phpunit configuration:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<phpunit
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
        bootstrap="vendor/autoload.php"
>
    <coverage>
        <include>
            <directory>src/</directory>
        </include>
        <report>
            <clover outputFile="build/logs/clover.xml"/>
        </report>
    </coverage>
    <testsuites>
        <testsuite name="FluidDocumentationGenerator">
            <directory>tests/</directory>
        </testsuite>
    </testsuites>
</phpunit>
```

Haven't the permissions to add commits to PRs.

linawolf commented:
@brotkrueml I added this repository to the Active Documentation Members, you should have Maintainer rights now

linawolf committed:
[BUGFIX] Annotate required arguments as required in RST

resolves https://github.com/TYPO3-Documentation/fluid-documentation-generator/issues/10

linawolf commented:
And applied the changes you suggested
